### ISSUE TITLE
The attention mask and the pad token id were not set. How do I fix it？

### ISSUE THREAD

**Issue by zoenders** [huchenlei/ComfyUI_omost#31]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.

**Comment by adsllk** [comment]
同样的问题，求解
